chain-api/src/types/DexDtos.ts

```diff
@@ -269,7 +272,8 @@ export class BurnDto extends ChainCallDTO {
     tickLower: number,
     tickUpper: number,
     amount0Min: BigNumber,
-    amount1Min: BigNumber
+    amount1Min: BigNumber,
+    positionId?: string
```

Comment:
this is a low-level constructor, so I'd recommend to pass all parameters explicitly:

```suggestion
    positionId: string | undefined
```

the property declaration should remain as is:

```
public positionId?: string;
```